### PR TITLE
fix(ui): clear search input after every search

### PR DESCRIPTION
**Pull request by rudolfs**
Fixes https://github.com/radicle-dev/radicle-upstream/issues/1771.

### CODE REVIEW

File: cypress/integration/search.spec.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+      // Make sure the search input is cleared after the search.
+      commands.pick("sidebar", "search").click();
+      commands.pick("search-input").should("have.value", "");
+    });
+
+    it("follows the project by pressing the [enter] hotkey", () => {
+      commands.pick("sidebar", "search").click();
+      commands.pasteInto(["search-input"], `rad:git:${projectId}`);
+      cy.wait(400);
```
Comment (geigerzaehler) [inline cypress/integration/search.spec.ts:30]: Could you add a comment explaining why we need this?

File: ui/Modal/Search.svelte [hunk 2]
```diff
@@ -53,7 +57,9 @@
     }
   };
   const follow = () => {
-    requestProject(value);
+    if ($urnValidation.status === ValidationStatus.Success) {
+      requestProject(value);
```
Comment (geigerzaehler) [inline ui/Modal/Search.svelte:61]: It’s probably worth adding a test for invalid URNs, either in this PR or as a follow-up

File: cypress/integration/search.spec.ts (hunk 1 above)
Comment (rudolfs) [reply cypress/integration/search.spec.ts]: That was actually a debug leftover, removed it here: 1f5137005d4dba1745956e587ce428d6733a3eef.

File: ui/Modal/Search.svelte (hunk 2 above)
Comment (rudolfs) [reply ui/Modal/Search.svelte:61]: 045faac2fd0ed352b4a2e8d9c0b65f3e12655356

**Review by geigerzaehler — APPROVED** [review]
:chains:

**Pull request merged**